src/collectd.conf.pod

```diff
@@ -5360,6 +5360,69 @@ refer to them from.
 
 =back
 
+=head2 Plugin C<ovs_events>
+
+The I<ovs_events> plugin monitors the link status of OVS connected interfaces,
+dispatches the values to collectd and send the notification whenever the link
+state change occurs. This plugin uses OVS DB to get a link state change
+notification.
+
+B<Synopsis:>
+
+ <Plugin "ovs_events">
+   OvsDbServerUrl "tcp:127.0.0.1:6640"
+   Interfaces "br0" "veth0"
+   SendNotification false
+ </Plugin>
+
+The plugin provides the following configuration options:
+
+=over 4
+
+=item B<OvsDbServerUrl> I<server>
```

Comment:
Please consider turning this into an option with two values, "node" and "service" (aka. "host" and "port"). That makes it much easier to support IPv6. See also my comments below.